force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -252,11 +337,19 @@ public with sharing class ClassesObjectsMethods {
      */
     public static String createAndGetBookDetails() {
         // Create a Book object with title "A Brief History of Time" and author "Stephen Hawking"
-
+        Book newbook  = new Book('A Brief History of Time' ,'Stephen Hawking');
+        newBook.Title = 'A Brief History of Time';
+        newBook.Author = 'Stephen Hawking';
+        
         // Call the getBookDetails method on the book object to get the details of the book
-
+        
+        String bookDetails = getBookDetails(newBook);
         // Return the details of the book
-        return null; // Replace null with the variable you used to store the result
+        return bookDetails; // Replace null with the variable you used to store the result
+    }
+    private static String getBookDetails(Book book) {
+        // Format the book details into a string
+        return 'Title: ' + book.Title + ', Author: ' + book.Author;
```

Comment:
This method does not really need an argument. It is meant to return its own instance properties. 

```
 public static String createAndGetBookDetails() {
        return new Book('A Brief History of Time', 'Stephen Hawking').getBookDetails();
    }
```